eth/main.cpp

```diff
@@ -350,14 +372,14 @@ int main(int argc, char** argv)
 	bool pinning = false;
 	bool enableDiscovery = false;
 	bool noPinning = false;
-	unsigned networkId = (unsigned)-1;
+	unsigned networkID = (unsigned)-1;
```

Comment:
Maybe make the `(unsigned)-1` to a typedef or an enum of uninitialized network ID for readability?